examples/with-schedules/README.md

```diff
@@ -0,0 +1,59 @@
+# EventBridge Lambda & Scheduler Schedules Example
```

Comment:
Please update the description a bit to match the reality of this example (compare with `with-lambda-scheduling`).